src/tests/Home.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from '../redux/store';
+import Home from '../pages/Home';
+
+test('renders correctly', () => {
+  const tree = render(
+    <Provider store={store}>
+      <BrowserRouter>
+        <Home />
+      </BrowserRouter>
+    </Provider>,
+  );
+
+  expect(tree).toMatchSnapshot();
```

Comment:
- [ ] Great job adding a `Snapshot` test @Errytagedesign, but as part of the project we need to have `unit tests for several components of the app and some pure functions in Redux code` and `integration tests using React Testing Library` Please add this tests to make sure the app is reliable. Remember `snapshot` test only asserts the layout remains the same.